### PR TITLE
feat: adding job seekers guide

### LINKED ISSUE #203: Feature: Create a Guide for Using OpenSauced for Job Seekers
### Type of feature

🍕 Feature

### Current behavior

We don't currently have any official guides on using OpenSauced.

### Suggested solution

Create a guide on using OpenSauced for Job Seekers. The guide should include:

1. An introduction to matching job requirements to skills
2. How to use each OpenSauced feature to relate to your job search
3. How to share your OpenSauced profile

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by jdwilkin4**
## Description

This PR is currently in draft status and will be responsible for adding a job seekres guide on how to use OpenSauced for boosting your job profile. 

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents
closes #203 

## Mobile & Desktop Screenshots/Recordings

Screenshot 2023-12-04 at 4 12 08 PM

## Added to documentation?

- [ ] 📜 README.md
- [x] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by jdwilkin4** [comment]
Hey @BekahHW ! 

Right now for this first draft, I wrote this guide all in one file.
But I didn't know if we wanted to break it up into multiple files all under the section I currently title job seekers guide.

Also, for this first draft, my initial approach was to use the open source table you gave me and craft the guide around that and make mention of the different features, throughout. 

Looking forward to your thoughts 👍

File: docs/job-seekers-guide/categorize-contributions.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
+---
+id: job-seekers-guide-how-to-organize-your-open-source-contributions
+title: "How to organize your open source contributions for jobs"
```
Comment (jdwilkin4) [inline docs/job-seekers-guide/categorize-contributions.md:3]: Also, not super in love with the title at the moment so I would be open to suggestions

Comment (adiati98) [reply docs/job-seekers-guide/categorize-contributions.md:3]: Maybe something like:

- Keeping Track Open Source Contributions for Job Seeking
- Organizing Open Source Contributions to Make Your Resume Stand Out
- Making Resume Stand Out through Open Source Contributions

Also @jdwilkin4, for consistency throughout the docs, let's capitalize all titles 😊

Comment (BekahHW) [reply docs/job-seekers-guide/categorize-contributions.md:3]: A couple more options

- "How to Effectively Track and Showcase Your Open Source Contributions for Career Advancement"
- "How to Build a Compelling Portfolio with Your Open Source Work"
- "How to Translate Open Source Contributions into Career Opportunities"
- "How to Document Your Open Source Journey for Professional Success"
- "How to Leverage Your Open Source Experience in Your Resume"

File: docs/job-seekers-guide/categorize-contributions.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+### Using your OpenSauced Dev Card for networking
+
+Your [OpenSauced Dev card](../features/dev-card.md) serves as your "open source business card" to showcase all of your recent open source activities. You will find recent activity concerning the projects you contribute to, frequency of pull requests and projects you are interested in.
+
+You can easily share your Dev card on social media platforms like Twitter and LinkedIn to gain attention from potential recruiters and hiring managers.
+
+![OpenSauced Dev card](../../static/img/a-dev-card.png)
+
+### Using the OpenSauced Connections feature
```
Comment (BekahHW) [inline docs/job-seekers-guide/categorize-contributions.md:156]: We've deprioritized this feature, so let's remove this section on Connections

**Comment by BekahHW** [comment]
This is looking awesome!!! I love the breakdown of the sections and the images balanced throughout.

A couple of suggestions:
1. In the Contribution tracking section, it might be worth adding highlights in there as a subheading. It's a way to track the contributions you're most proud of.
2. Also in the contribution tracking section, we could add lists and insights. Lists can be used to add people you interacted with during your contributions. This is a good way to keep track of people you can potentially use for references and referrals. Insights can be used to track the repositories you contributed to.
3. Other uses that could be integrated: insights can be used to track repositories of open source companies that are hiring or that you want to contribute to. Lists can be used to track other people in your network to reach out to about their contributions.

It would be useful to have a working example at the end. One that has the worksheet filled out. That's a lot to ask, so I can always do a second blog post with a filled out example if you'd prefer.

File: docs/job-seekers-guide/categorize-contributions.md (hunk 1 above)
Comment (CBID2) [reply docs/job-seekers-guide/categorize-contributions.md:3]: How about this one:

- “How to Leverage the Power of Open Source in your Job Search”
- “How to use Open Source in your Job Search”

**Comment by jdwilkin4** [comment]
Awesome, thanks for the feedback @BekahHW 

For the working example, should we just use dummy data or should the working example be based on a real person's contributions like myeself?

**Comment by BekahHW** [comment]
> Awesome, thanks for the feedback @BekahHW
> 
> For the working example, should we just use dummy data or should the working example be based on a real person's contributions like myeself?

I think either works. Or a combo.

**Review by BekahHW — APPROVED** [review]
I made a couple of minor additions (links to OpenSauced, move text around) but otherwise this looks great!

My one question, and I'd love to know what you think, @jdwilkin4, is whether we should have the chart at the beginning. I think it feels a little overwhelming. My thought would be to mention it in the beginning kind of like this:

> In this guide, we have created a table that you can use to categorize and keep detailed notes for each contribution you have made. We will break down each of these table components and talk about how to use this in your resume and job profile, and you can see a full example at the end of this guide.

What do you think?

**Comment by jdwilkin4** [comment]
@BekahHW 

Yeah, I agree with remove the full table at the beginning.
I can push up a commit now for that 👍

**Pull request merged**